platform-emf/src/main/java/com/softicar/platform/emf/form/EmfFormAttributesDiv.java

```diff
@@ -20,4 +22,22 @@ public EmfFormAttributesDiv(R tableRow, boolean editMode) {
 			}
 		}
 	}
+
+	//FIXME This works, but needs simplification
+	private boolean isScopeViewPermissionMissing(IEmfAttribute<R, ?> attribute) {
```

Comment:
I think we should rather show the attribute in any case, but then disable the button to open the view, i think this will be less confusing for users and easier to implement. it will also be rarely the case that someone cannot see the scope of an entity that they can see

Reply:
well the button already isnt appended if the user cant access it, see screenshot